internal/policies/apparmor/apparmor_test.go

```diff
@@ -22,58 +25,68 @@ var update bool
 func TestApplyPolicy(t *testing.T) {
 	t.Parallel()
 
-	defaultMachineProfile := []entry.Entry{{Key: "apparmor-machine", Value: "usr.bin.foo"}}
+	defaultMachineProfile := entry.Entry{Key: "apparmor-machine", Value: "usr.bin.foo"}
 
 	tests := map[string]struct {
 		entries []entry.Entry
 		user    bool
 
-		noParserOutput          bool
+		noEntries              bool
+		noParserOutput         bool
+		destAlreadyExists      string
+		readOnlyApparmorDir    string
+		noApparmorParser       bool
+		noLoadedPoliciesFile   bool
```

Comment:
Similarly, can we reuse existingLoadedPolicies with empty string instead of nil, or anything similar to avoid having that additional parameter? (It could be a special name too)